src/applications/disability-benefits/686c-674/config/chapters/674/student-last-term-information/studentLastTerm.js

```diff
@@ -53,7 +53,19 @@ export const uiSchema = {
       'ui:title': 'Last term school’s name',
     },
     schoolAddress: {
-      ...{ 'ui:title': 'Last term school’s address' },
+      ...{
+        'ui:title': 'Last term school’s address',
+        'ui:options': {
+          updateSchema: (formData, formSchema) => {
+            if (!formData.studentDidAttendSchoolLastTerm) {
+              return {
+                required: [],
+              };
+            }
+            return formSchema;
```

Comment:
This might look cleaner with one return:

```js
updateSchema: (formData, formSchema) =>
  formData.studentDidAttendSchoolLastTerm
    ? formSchema
    : { required: [] },
```

Reply:
True! I'll update 👍